### ISSUE TITLE
Options do not show in form preview

### ISSUE THREAD

**Issue by janagombitova** [akvo/akvo-flow#2592]
When testing 1.9.31 on UAT1 I found an issue that the options saved for a question do not show in the form preview. This issue is not happening on the live released and deployed version. 

Below are two screen shots showing the question has saved options, but they do not show in the preview. 

The issue happens for newly created survey forms as well as for forms that were created before this version was deployed. 

![screen shot 2018-04-18 at 11 08 07](https://user-images.githubusercontent.com/12456965/38922733-2915d66a-42f9-11e8-875c-3ad30e141d29.png)
<img width="849" alt="screen shot 2018-04-18 at 11 07 40" src="https://user-images.githubusercontent.com/12456965/38922734-2abec5e4-42f9-11e8-92c3-3cc177df0d74.png">

**Comment by stellanl** [comment]
We overlooked this use when we changed the Radio Buttons for the export range seletion.

**Comment by kymni** [comment]
Have reverted changes that were made in #2583 which should fix this issue once #2597 is merged into the release branch

**Comment by janagombitova** [comment]
Tested on UAT1 works well now.
